pkg/gadgets/trace/tcp/types/types.go

```diff
@@ -15,21 +15,43 @@
 package types
 
 import (
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	MountNsID uint64 `json:"mountnsid,omitempty"`
-	Pid       uint32 `json:"pid,omitempty"`
-	Comm      string `json:"comm,omitempty"`
-	IPVersion int    `json:"ipversion,omitempty"`
-	Saddr     string `json:"saddr,omitempty"`
-	Daddr     string `json:"daddr,omitempty"`
-	Sport     uint16 `json:"sport,omitempty"`
-	Dport     uint16 `json:"dport,omitempty"`
-	Operation string `json:"operation,omitempty"`
+	Operation string `json:"operation,omitempty" column:"t,width:1,fixed"`
+	Pid       uint32 `json:"pid,omitempty" column:"pid,width:7,fixed"`
+	Comm      string `json:"comm,omitempty" column:"comm,width:16,fixed"`
+	IPVersion int    `json:"ipversion,omitempty" column:"ip,width:2,fixed"`
+	Saddr     string `json:"saddr,omitempty" column:"saddr,width:22"`
+	Daddr     string `json:"daddr,omitempty" column:"daddr,width:22"`
```

Comment:
What do we want to do with the IP addresses? For IPv4 it should have a max=15 to cover this case: `192.168.112.123`. But, what about the min and IPv6?

```bash
$ ./kubectl-gadget trace tcp -n test
-ns
NODE      NAMESPACE POD       CONTAINER T PID     COMM             IP SADDR  DADDR  SPORT DPORT
worker    test-ns   test-pod  test-pod  X 200004  wget             4  192.1… 1.1.1… 51072 80
worker    test-ns   test…test test-pod… C 200008  wget             4  192.1… 1.1.1… 38380 80
```